patches/@parcel+bundler-default+2.9.1.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@parcel/bundler-default/lib/DefaultBundler.js b/node_modules/@parcel/bundler-default/lib/DefaultBundler.js
```

Comment:
These patches will go away once https://github.com/parcel-bundler/parcel/pull/9156 is merged and released.